configs/experiment/herwig_event_multihadron.yaml

```diff
@@ -67,13 +58,3 @@ model:
 
   num_critics: 1
   num_gen: 5
```

Comment:
Note for when we start tranining: in the previous paper, we have made one generator step for each critic step.